RELEASE_NOTES.md

```diff
@@ -1,5 +1,20 @@
+Release Notes for v5.3.0
+========================
+
+Note: If you are upgrading from an older version, it is advisable to upgrade
+incrementally, following the instructions in the previous versions' release
+notes.
+
+These notes are usually structured in 4 sections: **Migration Instructions**,
```

Comment:
It is empty for this release. Guess it makes more sense to exclude it. The approach in e.g. krill